eng/Versions.props

```diff
@@ -1,7 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <!-- We need to move to 4.7 as part of our versioning change when we switch to arcade to avoid downgrading versions -->
+    <MajorVersion Condition="'$(MajorVersion)' == ''">4</MajorVersion>
+    <MinorVersion Condition="'$(MinorVersion)' == ''">7</MinorVersion>
```

Comment:
Did you intend to move everything in the repo to 4.7?

Reply:
Everything except netstandard.dll, which is handled by https://github.com/dotnet/standard/pull/951/commits/caaa70c538c141bb904a246b008b41d6689512eb#diff-82d0524075c8f50a0573bb8342f78146R5